src/main/java/rx/schedulers/Schedulers.java

```diff
@@ -137,4 +138,52 @@ public static TestScheduler test() {
     public static Scheduler from(Executor executor) {
         return new ExecutorScheduler(executor);
     }
+    
+    /**
+     * Starts those standard Schedulers which support the SchedulerLifecycle interface.
+     * <p>The operation is idempotent and threadsafe.
+     */
+    public static void start() {
```

Comment:
We don't even start the Executor so there is nothing that would start it. We have used that approach in Hystrix for years, and there we used the `reset` model rather than shutdown/start: https://github.com/Netflix/Hystrix/blob/master/hystrix-core/src/main/java/com/netflix/hystrix/Hystrix.java#L42

If we completely shutdown the Executor, and then lazily recreate it, there is nothing to start threads, so it would be safe.